.github/workflows/pull-request.yml

```
@@ -36,6 +36,16 @@ jobs:
           docker compose logs --no-color mail > docker-compose-mail.log
       - name: Check the site
         run: docker run --network container:fosslight_web appropriate/curl -s --retry 3 --retry-connrefused http://localhost:8180/session/login
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)